src/lib/stable_structures/stable_b_tree_map.ts

```diff
@@ -1,24 +1,24 @@
 import { None, Opt, Some } from '../candid/types/constructed/opt';
 import { nat64 } from '../candid/types/primitive/nats/nat64';
 import { nat8 } from '../candid/types/primitive/nats/nat8';
-import { encode, decode } from '../candid/serde';
+import { decode } from '../candid/serde';
 
 export interface Serializable {
     toBytes: (data: any) => Uint8Array;
     fromBytes: (bytes: Uint8Array) => any;
 }
 
 export function StableBTreeMap<Key = any, Value = any>(
-    keyType: Partial<Serializable>,
-    valueType: Partial<Serializable>,
+    keySerializable: Serializable,
+    valueSerializable: Serializable,
```

Comment:
I'd recommend just `key` and `value`. The type tells you it's serializable. Just an idea though so just resolve this comment if you like it as is.

Reply:
key and value are already used inside of each method though